<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintHWDocuments</class>
 <widget class="QDialog" name="PrintHWDocuments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Print handwritten documents</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Orders</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eNumOrders">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPrintOrders">
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Acts of executed works &amp; warranty cards</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eNumGives">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPrintGives">
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Diagnostic acts</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eNumDiags">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bPrintDiags">
        <property name="text">
         <string>Print</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/icons/printer.png</normaloff>:/icons/icons/printer.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>eNumOrders</tabstop>
  <tabstop>bPrintOrders</tabstop>
  <tabstop>eNumGives</tabstop>
  <tabstop>bPrintGives</tabstop>
  <tabstop>eNumDiags</tabstop>
  <tabstop>bPrintDiags</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
